ibu-imager/hack/clean.sh

```diff
@@ -1,13 +1,27 @@
 #!/bin/bash
 
-sudo rm -rf /var/tmp/container_list.done \
-    /var/tmp/backup  && \
-sudo rm -f /usr/local/bin/prepare-installation-configuration.sh \
-    /usr/local/bin/installation-configuration.sh && \
-sudo systemctl disable installation-configuration.service && \
-sudo systemctl disable prepare-installation-configuration.service && \
-rm -f /etc/systemd/system/installation-configuration.service \
-    /etc/systemd/system/prepare-installation-configuration.service && \
-sudo podman rmi quay.io/alosadag/ibu-seed-sno0:oneimage --force && \
-sudo systemctl enable --now kubelet && \
-    sudo systemctl enable --now crio
+# 1) cleanup backup dir
+sudo rm -rf /var/tmp/container_list.done /var/tmp/backup
```

Comment:
The whole script would likely run as root, so sudo wouldn't be needed

Reply:
Gotcha, removed all `sudo` prefixes.